nekoyume/Assets/_Scripts/BlockChain/ActionRenderHandler.cs

```diff
@@ -604,21 +606,34 @@ private void ResponseHackAndSlash(ActionBase.ActionEvaluation<HackAndSlash> eval
                                 .DoOnError(e => Debug.LogException(e));
                         });
 
+                var simulator = new StageSimulator(
+                    new LocalRandom(eval.RandomSeed),
+                    States.Instance.CurrentAvatarState,
+                    eval.Action.foods,
+                    eval.Action.worldId,
+                    eval.Action.stageId,
+                    Game.Game.instance.TableSheets.GetStageSimulatorSheets(),
+                    Game.Game.instance.TableSheets.CostumeStatSheet,
+                    2
```

Comment:
It would be nice to have **constructorVersion** `2` constant somewhere. Also includes HackAndSlash, MimisbrunnrBattle  etc..

Reply:
@hskim881028 @ipdae Do you have any idea for naming? (e.g. `StageSimulator.ConstructorVersionV1000xx`)

Reply:
hmm.. I think the example is good, too. `ConstructorVersion1000xx`

Reply:
I added `StageSimulatorVersionv100025` and I'll move it to lib9c side in the next PR.